pkg/kube/client.go

```diff
@@ -555,9 +534,6 @@ func (c *client) GetMemberRoll() memberroll.MemberRollController {
 // RunAndWait starts all informers and waits for their caches to sync.
 // Warning: this must be called AFTER .Informer() is called, which will register the informer.
 func (c *client) RunAndWait(stop <-chan struct{}) {
-	// make sure to watch all namespaces if we're not using a MemberRollController
-	c.SetNamespaces(metav1.NamespaceAll)
-
```

Comment:
This is no longer needed, because the client (and underlying informer factories) are already configured with `NamespaceAll`. 

Previously, this was necessary because, while the factory was initially created with `NamespaceAll`, the list of namespaces was then immediately set to empty (via the `istioinformer.WithNamespaces()` call (see above)). Now, the latter is no longer the case, so there's no need to set the client back to `NamespaceAll` here.